localstack/constants.py

```diff
@@ -148,7 +148,7 @@
 TEST_AWS_ACCOUNT_ID = os.getenv("TEST_AWS_ACCOUNT_ID") or DEFAULT_AWS_ACCOUNT_ID
 TEST_AWS_ACCESS_KEY_ID = os.getenv("TEST_AWS_ACCESS_KEY_ID") or "test"
 TEST_AWS_SECRET_ACCESS_KEY = os.getenv("TEST_AWS_SECRET_ACCESS_KEY") or "test"
-TEST_AWS_REGION_NAME = os.getenv("TEST_AWS_REGION") or "us-east-1"
+TEST_AWS_REGION_NAME = os.getenv("TEST_AWS_REGION_NAME") or "us-east-1"
```

Comment:
These kind of changes can be very dangerous, because this is technically a breaking change. Is this env variable really only used internally and only used for testing? I guess so, but just to make sure... 😛

Reply:
Yes, also I had a discussion around this with @viren-nadkarni who helped me pointing this out.